### DISCUSSION TITLE
QGIS QFieldCloud Plugin not accepting password/login    (doesn't help changing pwd either ...)

### DISCUSSION THREAD

**Discussion by janulimoelv** [opengisch/QField#2457] [Q&A]
Hei I am surely just doing some stupid mistake but maybe someone can help...

After first times of getting "Login failed" I changed password on the QFieldCloud page but not helping either....

I am using 4.0.0-BETA15 of the plugin to test out Cloud functionality for QField...

QGIS is 3.20.3-Odense on a norwegian Windows 10 Pro

Deinstalling / Reinstalling of the Plugin did not work either...

BUT:     Sending me an invitasjon to a second email to create a "new" user worked out ...

![image](https://user-images.githubusercontent.com/49278348/138918570-7e7fe387-b658-4c38-b519-727971167b9b.png)

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@janulimoelv Hey, can you please try with the latest 4.0.0-BETA20? There are quite a lot of changes recently and probably it solves your issue?

**Comment by Iagree12** [comment]
Did you find a solution? I deleted and reinstalled the plug-in multiple times updated it, changed my password, tried a different mail, nothing worked.

**Reply by suricactus** [reply]
Can you share a little bit more details about you operating system, qgis version, qfieldsync version and qfieldcloud username?

**Reply by Iagree12** [reply]
Sure, operating system is Windows 10 Enterprise, qgis version 3.22.3, qfieldsync version is 4.00. The username is praktikum_natur.

**Reply by suricactus** [reply]
Do you manage to login using the web interface or QField? If you still fail to login, try to reset the password.

**Reply by Iagree12** [reply]
Yes I can login via the browser with no problem.

**Reply by suricactus** [reply]
Are you sure you entered the exact same characters as originally wanted? Can you please change your password, try again and share results? Make sure that capital letters matter.

**Reply by Iagree12** [reply]
You had me doubting my sanity for a moment, but yes, I am sure. It works fine in the browser and I can also sign in on other devices no problem. The difficulties are specific to this one PC which I am assigned to at work, would you recommend de-/reinstalling QGIS completely?

**Reply by suricactus** [reply]
Sorry for questioning you on basic things. Just wanted to make sure you have carefully written the password as it is. I have done obvious mistakes myself multiple times. No need to reinstall QGIS completely, you can try with a new QGIS profile. If not, try registering a new user on QFieldCloud and try again.

**Reply by Iagree12** [reply]
It´s all good, I was just so sure I had he right password. It doesn´t work with a new profile either, but it works on other devices, I will just change PCs for now, but thank you for your help.

**Reply by suricactus** [reply]
Sorry to hear that. One last question before giving up. If you go to QGIS menu and make sure that `View -> Panels -> Debugging & Development Tools` is open, can you try to login and check for a POST request to app.qfield.cloud/api/v1/auth/login/ . If you open the details, you should check the value on the `Content -> Data` line. It should have something like this:

![image](https://user-images.githubusercontent.com/2820439/164574382-5f3627df-99e3-4432-8782-3e85f510cb7d.png)

Can you make sure you see your username and password there? Note the value continues after the ellipsis.

No worries, this data is only visible on your QGIS. :)
